src/iop/basecurve.c

```diff
@@ -184,6 +184,8 @@ static const basecurve_preset_t basecurve_camera_presets[] = {
   {"Nikon D90", "NIKON CORPORATION", "NIKON D90", 0, 51200, {{{{0.000000, 0.000000}, {0.011702, 0.012659}, {0.122918, 0.289973}, {0.153642, 0.342731}, {0.246855, 0.510114}, {0.448958, 0.733820}, {0.666759, 0.894290}, {1.000000, 1.000000}}}, {8}, {m}}, 0, 1},
   // contributed by Pascal Obry
   { "Nikon D800", "NIKON", "D800", 0, 51200, { { { { 0.000000, 0.000000 }, { 0.001773, 0.001936 }, { 0.009671, 0.009693 }, { 0.016754, 0.020617 }, { 0.024884, 0.037309 }, { 0.048174, 0.107768 }, { 0.056932, 0.139532 }, { 0.085504, 0.233303 }, { 0.130378, 0.349747 }, { 0.155476, 0.405445 }, { 0.175245, 0.445918 }, { 0.217657, 0.516873 }, { 0.308475, 0.668608 }, { 0.375381, 0.754058 }, { 0.459858, 0.839909 }, { 0.509567, 0.881543 }, { 0.654394, 0.960877 }, { 0.783380, 0.999161 }, { 0.859310, 1.000000 }, { 1.000000, 1.000000 } } }, { 20 }, { m } }, 0, 1 },
+  {"Fujifilm X-E2S", "FUJIFILM", "X-E2S", 0, 51200, {{{{0.000000, 0.000000}, {0.004819, 0.001637}, {0.020269, 0.019512}, {0.040477, 0.056942}, {0.056822, 0.093118}, {0.085728, 0.163358}, {0.189989, 0.411522}, {0.293449, 0.582470}, {0.344385, 0.649834}, {0.578275, 0.814871}, {0.848233, 0.908421}, {1.000000, 1.000000}}}, {12}, {m}}, 0, 1},
```

Comment:
I'm going to merge everything but that commit. We don't take custom base/tone curves. And custom color matrices. Because we have no meant (and hands) to verify their validity.